src/test/java/org/gridsuite/study/server/service/ReportServiceTest.java

```diff
@@ -129,10 +128,15 @@ public Object getTypeId() {
             @NotNull
             public MockResponse dispatch(RecordedRequest request) {
                 String path = Objects.requireNonNull(request.getPath());
-                if (path.matches("/v1/reports/.*")) {
+                if (path.matches("/v1/reports/.*\\?defaultName=.*")) {
                     return new MockResponse().setResponseCode(HttpStatus.OK.value())
-                        .setBody(mapper.writeValueAsString(getNodeReport(Objects.requireNonNull(request.getRequestUrl()).pathSegments().get(2), request.getRequestUrl().queryParameter(QUERY_PARAM_REPORT_DEFAULT_NAME)).getSubReporters()))
-                        .addHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE);
+                            .setBody(mapper.writeValueAsString(getNodeReport(Objects.requireNonNull(request.getRequestUrl()).pathSegments().get(2), request.getRequestUrl().queryParameter(QUERY_PARAM_REPORT_DEFAULT_NAME)).getSubReporters()))
+                            .addHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE);
+                } else if (path.matches("/v1/subreports/.*\\?severityLevels=.*")) {
```

Comment:
Is it worth it to include `&` in these patterns, in case we have something like `/v1/reports/1234?hello=true&defaultName` ?

Reply:
I dont get this comment

Reply:
The current regex will only match if severityLevels is the first parameter in the URL. If it is the second, then it would match "&severityLevel=" instead of "?severityLevel="

Reply:
ok, I get it.
But we have only one param currently, so severityLevels is always first.